### DISCUSSION TITLE
logos with ligatures aren't displaying properly in powershell, cmd, or nushell on windows11

### DISCUSSION THREAD

**Discussion by daedalusman** [fastfetch-cli/fastfetch#1419] [General]
when i use fastfetch --print-logos, there are some logos that are not showing up correctly. I have attached a screenshot of one example. 
![Screenshot 2024-11-26 230841](https://github.com/user-attachments/assets/289345c4-ac3d-4c14-a059-2a005770cd0f)

They are displayed this way in powershell, cmd.exe, and nushell. I've tried in both wezterm and windows terminal, as well, and the same issue in both terminals. The weird thing is that at one point they did display correctly in cmd.exe but no longer do. 

Does anyone know how I can get this working correctly. 

Thanks for the help, hopefully we can get this solved!

**Comment by CarterLi** [comment]
Try:

1. `chcp 65001` before running fastfetch
2. change a different term font

**Reply by daedalusman** [reply]
Hey, thanks for the quick response!

I didn't know what chcp did so I looked it up. I was thinking font encoding might be the issue but wasn't sure how to check or change that. When I learned what that command did I was able to figure out some things to try out. My terminals were not using utf-8. But when I was searching around about this issue I found a setting in windows to set utf-8 globally. Once I set that option in windows, running chcp confirms that the code page is now in 65001, it was in 437. Now everything is working well it seems. 
![Screenshot 2024-11-27 210439](https://github.com/user-attachments/assets/a2e5b262-e87a-4fdf-aa09-844e95e6ea0a)


Again, thanks so much for the help! I learned some new things today, that's always a win.
